src/Schema/Endpoint.php

```diff
@@ -310,7 +310,7 @@ public function getAttribute(string $key, $default = null)
 
 	protected function generatePattern(): string
 	{
-		$rawPattern = $this->getAttribute('pattern', null);
+		$rawPattern = $this->getAttribute('pattern');
```

Comment:
@f3l1x Currently it's useless, as there is no `if (func_num_args() < 2) ...` check in `Endpoint::getAttribute()`, but maybe there should be?
I am not sure. We use these checks at some places. Personally, I don't like them. It require knowledge of how method works internally, signature is lying to user, that it returns null by default.